src/main/python/openresearch/retrieve_pubmed_dblp.py

```diff
@@ -0,0 +1,82 @@
+# -*- coding: utf-8 -*-
```

Comment:
Could you just have used openresearch/retrieve.py since the input and output files have the same format? We could just add a check to restrict the results if a candidates_file is passed (I would call this argument `valid_doc_ids` or something similar).

Reply:
yes that will make directory clean, i've changed that